compiler/rustc_mir/src/transform/check_consts/ops.rs

```diff
@@ -397,9 +397,13 @@ impl NonConstOp for PanicNonStr {
     }
 }
 
+/// Comparing raw pointers for equality.
+/// Not currently intended to ever be allowed, even behind a feature gate: operation depends on
+/// allocation base addresses that are not known at compile-time.
```

Comment:
Are there any other "ops" that should get a comment like this?

Reply:
well... there are similarly unallowable things. thread local statics, static mut, inline asm, ... they aren't really different, just more obvious?

Reply:
Hm yeah those are less subtle.